app/helpers/course/controller_helper.rb

```diff
@@ -46,4 +46,30 @@ def link_to_user(user, options = {}, &block)
       super(user, options, &block)
     end
   end
+
+  # Display the course_user_badge given the course and course_user
+  #
+  # @return [String] HTML string to render the course_user_badge
+  def display_course_user_badge
+    exp = current_course_user.experience_points
```

Comment:
@allenwq I added the logic at the view `app/views/layouts/course.html.slim`

```ruby
- unless current_course_user.staff?
  = display_course_user_badge
```

From your concern, I am thinking that we can pass `current_course_user` as a parameter into this method, so it is more explict. What do you think?